### ISSUE TITLE
how to install plpython3u in gp?

### ISSUE THREAD

**Issue by wonb168** [greenplum-db/gpdb#14309]
gp contain pl/python defalut, can install by `CREATE EXTENSION plpythonu;`
but it's python2, and how to install plpython3u?
My OS is centos7, gp is 6.12.

**Comment by yihong0618** [comment]
if you want to use plpython3u for gpdb6 for now, you can follow the steps:


# Install Python3 (take centos7 as example and python3.7)
yum install -y python37-devel

# Configure build environment to install at /usr/local/gpdb
PYTHON=/usr/bin/python3.7 ./configure --with-perl --with-python --with-libxml --prefix=/usr/local/gpdb

# Compile and install
make -j8
make -j8 install

then you can use 
```
CREATE LANGUAGE plpython3u;
CREATE FUNCTION import_succeed() returns text AS $$
  import sys
  print("hello world")
  return "succeeded, as expected"
$$ LANGUAGE plpython3u;
SELECT import_succeed();
```

**Comment by wonb168** [comment]
when I `make  -j8`
ERROR: `bison' is missing on your system. It is needed to create the
file `gram.c'. You can either get bison from a GNU mirror site
or download an official distribution of PostgreSQL, which contains
pre-packaged bison output.

**Comment by kainwen** [comment]
> when I `make -j8`
> 
> ```
> ERROR: `bison' is missing on your system. It is needed to create the
> file `gram.c'. You can either get bison from a GNU mirror site
> or download an official distribution of PostgreSQL, which contains
> pre-packaged bison output.
> ```

Please refer to README to know the steps to compile GPDB.

**Comment by wonb168** [comment]
`create extension plpython3u`
ERROR: could not access file "$libdir/plpython3": No such file or directory

**Comment by kainwen** [comment]
I think this plpython3 is intrduced by https://github.com/greenplum-db/gpdb/pull/13095 in 6X.

It is shipped since gpdb 6.22.0.

@beeender

**Comment by wonb168** [comment]
select version();
--PostgreSQL 9.4.26 (Greenplum Database 6.22.1 build commit:a15fdc9c3b2ed5b800b39dab4c7124e56266078c Open Source) on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 6.4.0, 64-bit compiled on Oct  5 2022 22:41:38

create language plpython3u;
ERROR: could not access file "$libdir/plpython3": No such file or directory

**Comment by beeender** [comment]
> It is shipped since gpdb 6.22.0.

But it is not a part of open source release.



To use the plpython3u, it has to be built by following steps in https://github.com/greenplum-db/gpdb/issues/14309#issuecomment-1286663545

**Please be noticed of the '--prefiix' arguments in the '.configure' command line, it specifies where 'make install' copies files to.**

`plpython3u` requires following files to be installed in the `$GPHOME`

```
├── lib
│   └── postgresql
│       └── plpython3.so
└── share
    └── postgresql
        └── extension
            ├── plpython3u--1.0.sql
            ├── plpython3u.control
            └── plpython3u--unpackaged--1.0.sql
```

Please check 1) if those files are created during build in `src/pl/plpython`, 2) if those files has been successfully installed to `$GPHOME`.

If there are created but not installed, two options to solve it:

1) Copy them manually to `$GPHOME`
or
2) correct the `--prefix` option in `.configure` command line and run `make install` again.
